### PR TITLE
[stable/graphite] adds Graphite 1.1.3 chart

### PR DESCRIPTION
**Pull request by monotek**
**What this PR does / why we need it**: Adds Graphite metrics server chart

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by costimuraru** [comment]
Nice!

File: incubator/graphite/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+[Graphite](https://graphiteapp.org/) is a monitoring tool.
+
+## Introduction
+
+This chart uses graphiteapp/graphite-statsd container to run Graphite inside Kubernetes.
+
+## Prerequisites
+
+- Has been tested on Kubernetes 1.7+
```
Comment (unguiculus) [inline incubator/graphite/README.md]: Won't install on 1.7. I'd suggest you go for 1.9+

File: incubator/graphite/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1beta2
```
Comment (unguiculus) [inline incubator/graphite/templates/deployment.yaml]: `apps/v1`?

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please move the chart to stable.

**Comment by monotek** [comment]
@unguiculus 
Thanks for the review :-)
Changed everything as requested.

**Comment by unguiculus** [comment]
I'd suggest you move the chart to stable.

**Comment by monotek** [comment]
Why? Isn't it normal to have new charts in the incubator first?
But i'm on vacation until next Tuesday anyway. So I could make the change in the middle of next week.

**Comment by unguiculus** [comment]
We tend to add charts to stable directly. I'd rather see incubator die. We don't need a place for have-baked stuff here. The chart looks pretty good. I don't see why you would want to add it to incubator.

**Comment by monotek** [comment]
@unguiculus Thanks for the info :-)
I moved graphite to the stable repo now.

P.s.: What about the other incubator charts?
Will these be moved in one go or does the chart owners need to do this one by one?

**Comment by monotek** [comment]
@unguiculus 
Imho we're ready to merge :-)

File: stable/graphite/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+## Prerequisites
+
+- Has been tested on Kubernetes 1.9+
+
+## Installing the Chart
+
+To install the chart with the release name `graphite`, run the following command:
+
+```bash
+$ helm install incubator/graphite --name graphite
```
Comment (davidkarlsen) [inline stable/graphite/README.md]: still references to incubator

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/graphite/README.md (hunk 3 above)
Comment (monotek) [reply stable/graphite/README.md]: thanks. fixed :-)

**Pull request merged**